test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -1398,7 +1398,7 @@ def forward(self, x):
     def test_isnan(self):
         class IsNaNModel(torch.nn.Module):
             def forward(self, input):
-                return torch.isnan(input)
+                return torch.isnan(input).byte()
```

Comment:
instead of add `.byte()`, remove the line https://github.com/pytorch/pytorch/blob/b22adeb0074f57c501422a6f44c5e0ad6f1c4adb/torch/onnx/symbolic_opset9.py#L1602